suite-native/module-send/src/components/OutputsReviewFooter.tsx

```diff
@@ -145,6 +172,27 @@ export const OutputsReviewFooter = ({
             }
 
             setTxid(sentTxid);
+        } else if (isRejected(sendResponse)) {
```

Comment:
Now this alert sheet will be displayed for any failed transaction. Not only those related to a SOL timeout. Please change this so the sheet is triggered only for the timeout. 

All the other states will be handled separately in: https://github.com/trezor/trezor-suite/issues/15464

Reply:
As we discussed, I'll leave it as is for now, but it will be probably redone in #15464 for better error handling.